### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by NathanBnm**
I just updated translations. But I built the app and translations aren't working at all. I don't see where is the issue.

### CODE REVIEW

**Comment by NathanBnm** [comment]
I've separated appdata translations from app translations for more clarity

File: src/WelcomeView.vala [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 public class SnippetPixie.WelcomeView : Gtk.Grid {
     construct {
-        var welcome = new Granite.Widgets.Welcome ( "Snippet Pixie", _("No snippets found."));
+        var welcome = new Granite.Widgets.Welcome ( "Snippet Pixie", _("No snippets found"));
```
Comment (ianmjones) [inline src/WelcomeView.vala]: The elementary standard (and my preference too) is to keep normal punctuation intact, so I'd prefer to keep the full stop.

File: po/extra/meson.build [hunk 2]
```diff
@@ -0,0 +1,7 @@
+i18n.gettext('extra',
+  args: [
+    '--directory=' + meson.source_root(),
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply po/extra/meson.build]: In fact it's not issued by this file, but I forgot to update them :wink: 

And sure you're right.

File: src/MainWindow.vala [hunk 3]
```diff
@@ -185,7 +185,7 @@ public class SnippetPixie.MainWindow : Gtk.ApplicationWindow {
 
         dialog.authors = {"Ian M. Jones"};
 
-        dialog.program_name = "Snippet Pixie";
+        dialog.program_name = _("Snippet Pixie");
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply src/MainWindow.vala]: I understand I'll remove it.

File: src/MainWindowHeader.vala [hunk 4]
```diff
@@ -66,7 +66,7 @@ public class SnippetPixie.MainWindowHeader : Gtk.HeaderBar {
         // pack_start (redo_button); // TODO: Add redo.
         pack_end (menu_button);
         // pack_end (search_entry); // TODO: Add search.
-        set_title ("Snippet Pixie");
+        set_title (_("Snippet Pixie"));
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply src/MainWindowHeader.vala]: Fixed too.

File: src/WelcomeView.vala (hunk 1 above)
Comment (NathanBnm) [reply src/WelcomeView.vala]: I find that the ending period on this specific element in the window looks very weird. It's like you're adding one to a title. I've never seen that in other apps, I'm not sure about this point :+1:

Comment (ianmjones) [reply src/WelcomeView.vala]: If you look at the welcome screen for apps like Music you'll see the full stop.

Comment (NathanBnm) [reply src/WelcomeView.vala]: Do you have a screenshot? I can't find this

Comment (ianmjones) [reply src/WelcomeView.vala]: Screenshot from 2019-03-29 19-31-50

Screenshot from 2019-03-29 19-33-02

File: src/MainWindow.vala [hunk 5]
```diff
@@ -186,7 +186,7 @@ public class SnippetPixie.MainWindow : Gtk.ApplicationWindow {
         dialog.authors = {"Ian M. Jones"};
 
         dialog.program_name = "Snippet Pixie";
-        dialog.copyright = _("Copyright © Byte Pixie Limited");
+        dialog.copyright = "Copyright © Byte Pixie Limited";
```
Comment (ianmjones) [inline src/MainWindow.vala]: This string should be translatable, even though the "Byte Pixie Limited" part of the string should remain intact.

File: src/WelcomeView.vala (hunk 1 above)
Comment (NathanBnm) [reply src/WelcomeView.vala]: Ok I understand. I still find that this is looking weird, but I'm going to fix this

File: src/MainWindow.vala (hunk 5 above)
Comment (NathanBnm) [reply src/MainWindow.vala]: I thought you were talking about that too. I'm gonna fix this.

**Comment by NathanBnm** [comment]
All should be good now :+1:

**Pull request merged**